channels/ni.m3u

```diff
@@ -5,3 +5,7 @@ https://cootv.cootel.com.ni:443/streams/d/SSLCH10-SD-ZUK/playlist.m3u8
 https://cootv.cootel.com.ni:443/streams/d/SSLCH11-SD-GZN/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTawbZLCedJzRX06CAyU-jX37n9iyTxMVcIfg&usqp=CAU" group-title="",CANAL 9
 https://cootv.cootel.com.ni:443/streams/d/SSLCH09-HD-CVS/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",VIVA NICARAGUA CANAL 13
+https://59802217c6db4.streamlock.net/ch13/videoch13/chunklist_w1505653761_DVR.m3u8
```

Comment:
Geo-blocked

Reply:
![image](https://user-images.githubusercontent.com/5430116/118645513-b2cc6480-b7e7-11eb-8af4-442cfeaa1519.png)
> Geo-blocked

I was able to get a single chunk without any VPN but something is wrong with https sertificate

Reply:
Yeah there's a certificate issue. I'm going to look at it.